### ISSUE TITLE
Rancher-images.txt -Make sure all images are mirrored to Rancher DockerHub account 

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#38785]
The following 2 images in the `rancher-images.txt` file are not from the Rancher's DockerHub account:
- busybox
- registry:2

We should get them mirrored (https://github.com/rancher/image-mirror) and then update their references in the codebase to use the new mirrored image.

**Comment by jiaqiluo** [comment]
Some information about the images

### busybox 
This image is used for collecting Rancher audit logs in Rancher's Helm chart, it is also hardcoded at multiple places in the Rancher codebase:
- https://github.com/rancher/rancher/blob/3e5fb4a537731df7ed96cc9ddae6a3902a57c8b1/chart/values.yaml#L25-L27
- https://github.com/rancher/rancher/blob/3e5fb4a537731df7ed96cc9ddae6a3902a57c8b1/chart/templates/deployment.yaml#L193
- https://github.com/rancher/rancher/blob/3e5fb4a537731df7ed96cc9ddae6a3902a57c8b1/pkg/image/export/main.go#L230
- https://github.com/rancher/rancher/blob/3e5fb4a537731df7ed96cc9ddae6a3902a57c8b1/pkg/image/resolve.go#L133


### registry:2
This images is used by the deprecated pipeline feature. 

- https://github.com/rancher/rancher/blob/3e5fb4a537731df7ed96cc9ddae6a3902a57c8b1/pkg/image/export/main.go#L230
- https://github.com/rancher/rancher/blob/3e5fb4a537731df7ed96cc9ddae6a3902a57c8b1/pkg/apis/management.cattle.io/v3/tools_system_images.go#L18

**Comment by jiaqiluo** [comment]
### Root cause
The following two images in the rancher-images.txt file are not from the Rancher's DockerHub account:
- busybox
- registry:2

### What was fixed, or what changes have occurred
Switch to using the rancher mirrored images and other necessary changes to make it work.

### Areas or cases that should be tested
Area 1: Rancher HA chart 
steps:
-  enable the `API Audit Log` when installing the Rancher Helm chart, see [rancher docs](https://rancher.com/docs/rancher/v2.6/en/installation/install-rancher-on-k8s/chart-options/#api-audit-log)  

expected result: 
- the mirrored image should be used in the `rancher--audit-log` container 
  
Area 2: Pipeline 
We can skip it because the feature Pipeline has been deprecated since Rancher 2.5.0; I do not think anyone can hit any issue around this image. 

### What areas could experience regressions?

- Rancher Helm chart 
- Pipeline feature   


### Are the repro steps accurate/minimal?
yes

**Comment by jiaqiluo** [comment]
Note:  This was merged before branching.

**Comment by sowmyav27** [comment]
The images in rancher-images.txt in 2.6.9-rc1 are
```
rancher/mirrored-library-busybox:1.31.1
rancher/mirrored-library-busybox:1.32.1
rancher/mirrored-library-busybox:1.34.1
rancher/mirrored-library-registry:2.7.1
```
whereas the images on 2.6.8 are
```
registry:2
busybox
rancher/mirrored-library-busybox:1.31.1
rancher/mirrored-library-busybox:1.32.1
rancher/mirrored-library-busybox:1.34.1
```

**Comment by markusewalker** [comment]
**QA TEST PLAN**
Scenario 1 - Compare/contrast images with Docker Hub to `rancher-images.txt`
1. Download the `rancher-images.txt` file and grep the images for busybox and registry.
     - Take note of all of the images
2. Navigate to Docker Hub and look at the Rancher images for busyboxy and registry.
3. Compare/contrast the images found in the `rancher-images.txt`; validate that they are the same.
_____
Scenario 2 - Installing Rancher with API Audit Log enabled
1. Setup Rancher with the flag `--set auditLog.level=1`, login as that user.
2. In the `rancher-audit-log`, validate that the appropriate mirrored images found in `rancher-images.txt` appear.

**Comment by markusewalker** [comment]
Validated that this is addressed in `v2.6.9-rc1`.

**ENVIRONMENT DETAILS**
- Rancher install: Helm
- Rancher version: `v2.6.9-rc1`
- Browser: Chrome	

**TEST RESULT
PASS**

**VERIFICATION STEPS**
**Scenario 1 - Compare/contrast images with Docker Hub to `rancher-images.txt`**
1. Downloaded the `rancher-images.txt` file and grepped the list for busybox and registry.
```
rancher/mirrored-library-busybox:1.31.1
rancher/mirrored-library-busybox:1.32.1
rancher/mirrored-library-busybox:1.34.1
rancher/mirrored-library-registry:2.7.1
```
2. Navigated to Docker Hub and looked for the busybox and registry images.
3. Compared/contrasted the images found in the rancher-images.txt; validated that they are the same.
_________
**Scenario 2 - Installing Rancher with API Audit Log enabled**
1. Setup Rancher with the flag `--set auditLog.level=2` and logged in
2. Navigated to Cluster Explorer > local > Apps > Installed Apps > rancher.
3. Validated that the `busyboxImage` is there and that it is using the correct image tag:
![image](https://user-images.githubusercontent.com/52591819/191384777-e09f6b01-2cf9-4b15-9c45-5aa764dff5b5.png)
